study-builder/studies/brugada/study-base.conf

```diff
@@ -0,0 +1,57 @@
+{
+  "tenant": {
+    "domain": ${auth0.domain},
+    "mgmtClientId": ${auth0.mgmtClientId},
+    "mgmtSecret": ${auth0.mgmtSecret},
+  },
+
+  "umbrella": {
+    "name": "Brugada Project",
+    "guid": "brugada",
+  },
+
+  "study": {
+    "name": "Brugada Project",
+    "guid": "brugada",
+    "studyEmail": ${studyEmail},
+    "baseWebUrl": ${baseWebUrl},
+    "irbPassword": ${irbPassword},
+    "plusCodePrecision": null,
+    "shareParticipantLocation": false,
+  },
+
+  "clients": [
+    {
+      "id": ${auth0.clientId},
+      "secret": ${auth0.clientSecret},
+      "passwordRedirectUrl": ${passwordRedirectUrl},
+    },
+  ],
+
+  "adminUser": {
+    "guid": "BRUGADAADMINUSER",
+  },
+
+  "supportedLanguages": [
+    {
+      "language": "en",
+      "name": "English",
+      "isDefault": true,
+    },
+  ],
+
+  "studyDetails": [
+    {
+      "language": "en",
+      "name": "Brugada Project",
+      "summary": null,
+    },
+  ],
+
```

Comment:
Please add settings so that we can add google analytics token
  "settings": {
    "analyticsEnabled": true,
    "analyticsToken": ${analyticsToken}
  },

Reply:
added